src/software/ai/hl/stp/evaluation/robot.h

```diff
@@ -46,5 +45,9 @@ namespace Evaluation
      * @param timestamp The time at which we want to know if the robot had the ball
      * @return True if the ball is close to the front dribbler and false otherwise
      */
-    bool robotHasPossession(Ball ball, Robot robot, Timestamp timestamp = Timestamp());
+    bool robotHasPossession(const Ball& ball, const Robot& robot,
+                            Timestamp timestamp = Timestamp());
+
+    bool robotBeingPassedTo(const World& world, const Robot& robot,
+                            Timestamp timestamp = Timestamp());
```

Comment:
We should add a javadoc for this function

Reply:
Still missing a javadoc here but this is marked as resolved?

Reply:
actually added it this time, oops

Reply:
Added a javadoc this time